OpenSim/Common/Test/testComponentInterface.cpp

```diff
@@ -1987,11 +2009,14 @@ void writeTimeSeriesTableForInputConnecteeSerialization() {
                                    dataFileNameForInputConnecteeSerialization);
 }
 
-void testListInputConnecteeSerialization() {
+TEST_CASE("Component Interface List Input Connectee Serialization Behavior")
+{
     // We build a model, store the input connectee paths, then
     // recreate the same model from a serialization, and make sure the
     // connectee paths are the same.
 
+    writeTimeSeriesTableForInputConnecteeSerialization();
```

Comment:
Was this added intentionally? Was the test not working before?

Reply:
The original code placed the call in `main` before running those two tests.

The test content is dependent on the `sto` file generated by `writeTimeSeriesTableForInputConnecteeSerialization`, but the original version of the code didn't encode it into each case (instead, it relied on tests being ran in a certain order).